src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```

Comment:
**Changes Required**
- [ ] The `App Component` should be a `Class Component` as described in the project requirement. Let us try and refactor accordlingly.

Reply:
Thank you for your comment, Solved as you can see in the image below
![Capture](https://user-images.githubusercontent.com/60292761/133386067-36b3fd7a-eb62-4f47-810e-b32bc70eef6e.PNG)